content/sensu-go/6.2/observability-pipeline/observe-schedule/metrics.md

```diff
@@ -0,0 +1,523 @@
+- The check must include the [`output_metric_format` attribute][10] with a value that specifies one of Sensu's [supported output metric formats][9].
+
+When a check includes correctly configured `command` and `output_metric_format` attributes, Sensu will extract the specified metrics from the check output and add them to the event data in the [metrics attribute][5].
+
+You can also configure the check `output_metric_handlers` attribute to use a Sensu handler that is equipped to handle Sensu metrics.
+See the [check reference][3] or [InfluxDB handler guide][23] to learn more.
+
+### Supported output metric formats
+
+Sensu supports the following formats for check output metric extraction.
+
+| Graphite           |      |
+---------------------|------
+output_metric_format | `graphite_plaintext`
+documentation        | [Graphite Plaintext Protocol][14]
+example              | {{< code plain >}}local.random.diceroll 4 123456789{{< /code >}}
+
+| InfluxDB           |      |
+---------------------|------
+output_metric_format | `influxdb_line`
+documentation        | [InfluxDB Line Protocol][15]
+example              | {{< code plain >}}weather,location=us-midwest temperature=82 1465839830100400200{{< /code >}}
+
+| Nagios             |      |
+---------------------|------
+output_metric_format | `nagios_perfdata`
+documentation        | [Nagios Performance Data][13]
+example              | {{< code plain >}}PING ok - Packet loss = 0%, RTA = 0.80 ms | percent_packet_loss=0, rta=0.80{{< /code >}}
+
+| OpenTSDB           |      |
+---------------------|------
+output_metric_format | `opentsdb_line`
+documentation        | [OpenTSDB Data Specification][16]
+example              | {{< code plain >}}sys.cpu.user 1356998400 42.5 host=webserver01 cpu=0{{< /code >}}
+
+| Prometheus         |      |
+---------------------|------
+output_metric_format | `prometheus_text`
+documentation        | [Prometheus Exposition Text][17]
+example              | {{< code plain >}}http_requests_total{method="post",code="200"} 1027 1395066363000{{< /code >}}
+
+## Enrich metrics with tags
+
+[Output metric tags][1] are custom tags you can apply to enrich the metric points produced by check output metric extraction.
+
+Use output metric tags for the output metric formats that do not natively support tags: Graphite Plaintext Protocol and Nagios Performance Data.
+
+Values for output metric tags are passed through to the metric points produced by check output metric extraction for formats that natively support tags (InfluxDB Line Protocol, OpenTSDB Data Specification, and Prometheus Exposition Text).
+
+You can use [check token substitution][22] for the [value attribute][21] to include any event attribute in an output metric tag.
+For example, this tag will list the `event.time` attribute:
+
+{{< code shell >}}
+"output_metric_tags": [
+  {
+    "name": "instance",
+    "value": "{{ .time }}"
```

Comment:
This example could be better.  I'd use something like `{{.entity.system.hostname}}` if you're going to have `instance` as the tag name.